README.md

```diff
@@ -1,72 +1,72 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Catalog of my things
 
-> Description the project.
+> This app help you to keep a record of different types of things you won: books, music albums, movies, and games. 
 
+# Project Presentation
+[Video](https://drive.google.com/file/d/1ZAu8zxPNCcYqfgXCJnXbAUk7_fXEQkg1/view)
```

Comment:
_Remember the presentation is 5 minutes maximum_

- [ ] Please re-record a new video not more than 5 mins long 🙏